documents/TestSuite/util/closure_color_scene.xml

```diff
@@ -0,0 +1,33 @@
+<World>
+   <Camera eye="0, 100, 300" look_at="0,0,0" fov="70" />
+
+   <!-- Background clear 
+   %background_color% : is the background color
+   -->
+   <ShaderGroup>
+      color Cin %background_color%;
+      shader constant_color_background layer1;
+   </ShaderGroup>
+   <Background resolution="1024" />
+   
+   <!-- Emitter -->
+   <ShaderGroup>
+      float power 326000;
+      shader emitter layer1;
+   </ShaderGroup>
+   <Quad corner="-100,100,0" edge_x="0,0,200" edge_y="200,0,0" is_light="yes"/>
+
+   <!-- Shader graph for routing to output layer:
+   %input_shader_type% : type of an input shader to feed into the output shader.
```

Comment:
This part is the same as for constant shading but the output shader is a closure color passthrough instead of an emission shader.